addon/webextension/_locales/en_US/messages.json

```diff
@@ -1,33 +1,79 @@
 {
```

Comment:
Should this file be in the source at all?  Isn't it generated from the properties file?